grc/gui/VariableEditor.py

```diff
@@ -246,81 +222,43 @@ def _handle_value_edited_cb(self, cell, path, new_text):
         Actions.VARIABLE_EDITOR_UPDATE()
 
     def handle_action(self, item, key, event=None):
-        """
-        Single handler for the different actions that can be triggered by the context menu,
-        key presses or mouse clicks. Also triggers an update of the flow graph and editor.
-        """
         if key == self.ADD_IMPORT:
             self.emit('create_new_block', 'import')
         elif key == self.ADD_VARIABLE:
             self.emit('create_new_block', 'variable')
         elif key == self.OPEN_PROPERTIES:
-            # TODO: This probably isn't working because the action doesn't expect a parameter
-            # Actions.BLOCK_PARAM_MODIFY()
-            pass
+            print("Opening properties dialog")
```

Comment:
Don't add `print` messages to GUI code. If you need to log something, use the logging infrastructure.